src/services/llms/openai.py

```diff
@@ -48,7 +48,7 @@ def get_ai_response(self, prompt: str) -> List[Dict[str, str]]:
             response = self.client.chat.completions.create(
                 model=self.model,
                 messages=[
-                    {"role": "system", "content": "You are a code review assistant."},
+                    {"role": "system", "content": "You are an expert code reviewer. You provide clear, actionable feedback to improve code quality, security, and performance. You communicate in the specified language while maintaining technical accuracy."},
```

Comment:
The change in the system prompt is quite verbose, which might lead to inconsistencies in the response length, potentially affecting performance. Consider simplifying the prompt to ensure better response times and efficiency.